azuread/table_azuread_application.go

```diff
@@ -93,13 +90,6 @@ func listAdApplications(ctx context.Context, d *plugin.QueryData, _ *plugin.Hydr
 		input.Filter = strings.Join(filter, " and ")
 	}
 
-	limit := d.QueryContext.Limit
```

Comment:
This seems like it may still be useful, i.e., a user asks for only 20 results, so we should only ask the Graph API for 20 results. Is there a reason we shouldn't have this code in?